models/mpas_atm/MPAS_model_modules/get_coeff_mod.f90

```diff
@@ -15,10 +10,9 @@ module get_coeff_mod
   public :: mpas_rbf_interp_func_3D_plane_vec_const_dir_comp_coeffs
 
 ! version controlled file description for error handling, do not edit
-character(len=256), parameter :: source   = &
-   "$URL$"
-character(len=32 ), parameter :: revision = "$Revision$"
-character(len=128), parameter :: revdate  = "$Date$"
+character(len=256), parameter :: source   = 'get_coeff_mod'
+character(len=32 ), parameter :: revision = ""
+character(len=128), parameter :: revdate  = ""
```

Comment:
Here we can remove revision and revdate entirely, like is done in the other files in this PR. Also remove them from the call to the error handler on 262

Reply:
done in 381b40eeaa98532c2bfb172b88eaa0d02d87cf75